controllers/apply-next/views/eligibility.njk

```diff
@@ -0,0 +1,58 @@
+{% extends "layouts/main.njk" %}
+{% from "components/form-fields/macros.njk" import inputChoice with context %}
+
+{% block title %}Eligibility | {{ formTitle }} | {% endblock %}
+
+{% block content %}
+    <main role="main" id="content">
+        <div class="content-box u-inner u-inner-wide-only">
+            <div class="s-prose u-constrained-wide">
```

Comment:
Wrapping this with `s-prose` will make form fields look a little wonky, might want to scope this a bit more, or remove it.

Reply:
👍 (was a copy/paste job...)